.gitignore

```diff
@@ -0,0 +1,13 @@
+node_modules/
```

Comment:
Would be nice to also add `.idea/` so we ignore JetBrains specific files.
Same for `.vscode`.

Reply:
That should be in your local global gitignore, not in the repository :)